src/app/GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -423,6 +423,13 @@ Y   Draw non relatives gray
                 IsCheckedFunc = () => AppSettings.ShowRevisionGridGraphColumn
             },
             new MenuCommand
+            {
+                Name = "showGitNotesColumnToolStripMenuItem",
+                Text = "Show git &notes column",
```

Comment:
Git is a name and should be capital
Not set in e.g. Repository-Maintenance
```suggestion
                Text = "Show Git &notes column",
```